internal/config/application.go

```diff
@@ -38,6 +38,7 @@ type Application struct {
 	Package           Packages       `yaml:"package" json:"package" mapstructure:"package"`
 	FileMetadata      FileMetadata   `yaml:"file-metadata" json:"file-metadata" mapstructure:"file-metadata"`
 	Secrets           Secrets        `yaml:"secrets" json:"secrets" mapstructure:"secrets"`
+	Contents          contents       `yaml:"contents" json:"contents" mapstructure:"contents"`
```

Comment:
Open question @luhring @alfredodeza : I think I should rename this from `contents` to `file-contents` to be consistent with `file-metadata` and another cataloger added in another PR. Any votes / thoughts here?

Reply:
I think that makes sense

Reply:
just made the adjustment